.github/workflows/workflow.yml

```
@@ -101,6 +115,9 @@ jobs:
       # disabled pending strategy discussion on exposing tunables
       # - uses: Determinatesystems/flake-checker-action@main
       - uses: DeterminateSystems/determinate-nix-action@v3
+        with:
+          extra-conf: |
```

actionlint (1.7.11)

[error] got unexpected character '?' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)